Templates/BlobTrigger-FSharp/metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "BlobTrigger",
+    "description": "$BlobTriggerFSharp_description",
```

Comment:
add $BlobTriggerFSharp_description to the resource file
https://github.com/Azure/azure-webjobs-sdk-templates/blob/dev/Resources/Resources.resx